docs/alerts.md

```diff
@@ -0,0 +1,40 @@
+## How do Utilization Alerts work?
+
+Utilization alerts work in a way that surprises some users, typically wondering
+why a specific utilization alert has not been notified. Here is a description of
+how they work so that users can understand why that is the case.
+
+### Utilization percentages
+
+Apisonator will emit alerts to the administrative instance for traffic crossing
+the following utilization percentage of an application _if configured to do so_:
+
+0, 50, 80, 90, 100, 120, 150, 200, 300
+
+However, only one alert percentage is to be notified in any given 24h period.
+
+### Alert notification
+
+Alerts are only sent **once** in any given 24 hours period for each application and
+utilization percentage regardless of the period for which the utilization
```

Comment:
We should also mention metrics.

When an alert has been sent for a given app and utilization %, it won't be sent again when going over limits for a different metric.

Reply:
@davidor does the change in the commit below make it all read ok to you?

Reply:
:+1: